internal/test_cases/utils.go

```diff
@@ -0,0 +1,57 @@
+package testcases
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/codecrafters-io/tester-utils/executable"
+)
+
+// CreateTemp creates a new temporary file in the directory dir with a name beginning with prefix
+// Cleaning the file is the responsibility of the caller
+// As dir is empty, it will create the file in the default temp directory
+func createTempFileWithContents(contents string) (string, error) {
+	tmpFileName := "test.lox"
+	if _, err := os.Stat(tmpFileName); err == nil {
+		err := os.Remove(tmpFileName)
+		if err != nil {
+			return "", fmt.Errorf("CodeCrafters internal error. Error removing tmp file: %v", err)
+		}
+	}
+
+	tmpFile, err := os.Create(tmpFileName)
+	if err != nil {
+		return "", fmt.Errorf("CodeCrafters internal error. Error creating tmp file: %v", err)
+	}
+	_, err = tmpFile.WriteString(contents)
+	if err != nil {
+		return "", fmt.Errorf("CodeCrafters internal error. Error writing to tmp file: %v", err)
+	}
+	err = tmpFile.Close()
+	if err != nil {
+		return "", fmt.Errorf("CodeCrafters internal error. Error closing tmp file: %v", err)
+	}
+
+	return tmpFile.Name(), nil
+}
+
+func getOutputFromStdOut(result executable.ExecutableResult) []string {
```

Comment:
"StdOut" here is capitalized, the underlying variable `result.Stdout` is not - inconsistent

Reply:
Name is also weird, sure we can do something better here - like `getStdoutLinesFromExecutableResult`, `getStderrLinesFromExecutableResult`